.golangci.yml

```diff
@@ -9,7 +9,6 @@ linters:
     - dogsled
     - exportloopref
     - errcheck
-    - goconst
```

Comment:
do we wanna pull this out in a separate commit @ main?

Reply:
I think it can be handled as part of the larger linter story, this was flagged as "remove" in @chatton's hackmd analysis of linters, I only removed because it started whinging at me here for reusing string literals in test vars